patch/etc/baseline/module-list.txt

```diff
@@ -42,7 +42,6 @@
 /com/google/http-client/google-http-client/main/google-http-client-jackson2-1.22.0.jar
 /com/google/inject/extensions/3.0/guice-assistedinject-3.0.jar
 /com/google/inject/guice/3.0/guice-3.0.jar
-/com/google/inject/guice/main/guice-4.0.jar
```

Comment:
Sure that we don't need guice-4.0 any more? In which case we could get rid of the slot

Reply:
This seems to be caused by some previous commit. I do not think I should solve that in the fix for 2033. I can add a separate commit for that, if you like?